wtpython/__main__.py

```diff
@@ -29,34 +31,38 @@ def parse_arguments() -> tuple[dict, list]:
     )
 
     flags, args = parser.parse_known_args()
+
+    # Set sys.argv as the intended script would receive them
+    sys.argv = ['python', *args[1:]]
     return vars(flags), args
 
 
 def main() -> None:
     """Run the application"""
     flags, args = parse_arguments()
-    tb = run_and_get_stderr(args)
-    if not tb:
-        return
+    exc = None
+
+    try:
+        runpy.run_path(args[0], run_name='__main__')
+    except Exception as e:
+        exc = e
+        error = ''.join(traceback.format_exception_only(type(e), e)).strip()
 
```

Comment:
Have you removed the check to see if `exc` is None? We can exit early in this case. No point trying to call the StackOverflow API

Reply:
I had that in there, not sure where it went, but it's back now.